components/rating/stories/template.js

```diff
@@ -88,3 +100,21 @@ export const Template = ({
 		</div>
 	`;
 };
+
+export const RatingSizeGroup = (args, context) => Container({
```

Comment:
The Sizes utility does the same thing but scales for you if the arg options change. Check out the Accordion story for an example:

```
export const Sizing = (args, context) => Sizes({
	Template,
	withBorder: false,
	withHeading: false,
	items: content,
	...args,
}, context);
```